Detectors/ITSMFT/ITS/simulation/src/V3Layer.cxx

```diff
@@ -524,11 +526,21 @@ TGeoVolume *V3Layer::createModuleInnerB(Double_t xmod, Double_t ymod, Double_t z
 {
   Double_t xtot, ytot, ztot, zchip;
   Double_t xpos, ypos, zpos;
-  char volumeName[30];
+  Bool_t dummyChip;
+  char chipName[30], sensName[30], volumeName[30];
```

Comment:
I suggest using `std::string` here, then it's safe even if the return of `GeometryTGeo::getITSChipPattern()` changes